core/tauri/src/menu/plugin.rs

```diff
@@ -0,0 +1,637 @@
+// Copyright 2019-2023 Tauri Programme within The Commons Conservancy
```

Comment:
this file is kinda big, can we move the commands to their own module?

Reply:
the commands make most of the file, so moving them into another module will still be a big file